eduvpn/actions/select.py

```diff
@@ -42,7 +43,7 @@ def select_profile(builder, verifier, lets_connect):
     if not meta:
         logger.info("no configuration selected, showing main logo")
         notebook.set_current_page(0)
-        return
+        return None
```

Comment:
in principe maakt dit niks uit, een lege return returned all `None`

Reply:
Dit veroorzaakte een probleem met de type casting.